### PR TITLE
Created README_hd.md

### PR DESCRIPTION
**Pull request by AkshitGulyan**
A Hindi Translation for README

# What does this PR do?
It adds the Hindi Translation for the README File !

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Can you also add the proper info in [this dictionary](https://github.com/huggingface/transformers/blob/7a1c68a8454c25c55f3f8978c182ea90e3412f5c/utils/check_copies.py#L39) so this translation is kept up to date when we add new models?

**Comment by AkshitGulyan** [comment]
added the proper info in [this dictionary](https://github.com/huggingface/transformers/blob/7a1c68a8454c25c55f3f8978c182ea90e3412f5c/utils/check_copies.py#L39)

By a new pull request 
[Update check_copies.py #19905](https://github.com/huggingface/transformers/pull/19905)

**Comment by sgugger** [comment]
No this should all be in the same pull request please.

**Comment by AkshitGulyan** [comment]
> No this should all be in the same pull request please.

Updated the check_copies.py in this current Pull Request and closed the previous Pull Request !

File: README_hd.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+### पाइप का उपयोग करना
+
+इस रिपॉजिटरी का परीक्षण Python 3.6+, Flax 0.3.2+, PyTorch 1.3.1+ और TensorFlow 2.3+ के तहत किया गया है।
+
+आप [वर्चुअल एनवायरनमेंट] (https://docs.python.org/3/library/venv.html) में 🤗 ट्रांसफॉर्मर इंस्टॉल कर सकते हैं। यदि आप अभी तक पायथन के आभासी वातावरण से परिचित नहीं हैं, तो कृपया इसे [उपयोगकर्ता निर्देश] (https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) पढ़ें।
+
+सबसे पहले, पायथन के उस संस्करण के साथ एक आभासी वातावरण बनाएं जिसका आप उपयोग करने और उसे सक्रिय करने की योजना बना रहे हैं।
+
+फिर, आपको Flax, PyTorch या TensorFlow में से किसी एक को स्थापित करने की आवश्यकता है। अपने प्लेटफ़ॉर्म पर इन फ़्रेमवर्क को स्थापित करने के लिए, [TensorFlow स्थापना पृष्ठ](https://www.tensorflow.org/install/), [PyTorch स्थापना पृष्ठ](https://pytorch.org/get-started /locally/# देखें) start-locally) या [Flax install page](https://github.com/google/flax#quick-install).
```
Comment (pacman100) [inline README_hd.md:207]: `install page` is inconsitent across framework links

File: README_hd.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+जब इनमें से कोई एक बैकएंड सफलतापूर्वक स्थापित हो जाता है, तो ट्रांसफॉर्मर निम्नानुसार स्थापित किए जा सकते हैं:
+
+`` `बाश
+पाइप स्थापित ट्रांसफार्मर
```
Comment (pacman100) [inline README_hd.md:212]: original command in English?

File: README_hd.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+यदि आप उपयोग के मामलों को आज़माना चाहते हैं या आधिकारिक रिलीज़ से पहले नवीनतम इन-डेवलपमेंट कोड का उपयोग करना चाहते हैं, तो आपको [सोर्स से इंस्टॉल करना होगा](https://huggingface.co/docs/transformers/installation#installing-from- स्रोत)।
+
+### कोंडा का उपयोग करना
+
+ट्रांसफॉर्मर संस्करण 4.0.0 के बाद से, हमारे पास एक कोंडा चैनल है: `हगिंगफेस`।
+
+ट्रांसफॉर्मर कोंडा के माध्यम से निम्नानुसार स्थापित किया जा सकता है:
+
+``` खोल स्क्रिप्ट
+कोंडा इंस्टाल-सी हगिंगफेस ट्रांसफॉर्मर
```
Comment (pacman100) [inline README_hd.md:224]: command in English?

File: README_hd.md [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+
+# भावना विश्लेषण पाइपलाइन का उपयोग करना
+>>> classifier = pipeline('sentiment-analysis')
+>>> classifier('We are very happy to introduce pipeline to the transformers repository.')
+[{'label': 'POSITIVE', 'score': 0.9996980428695679}]
+```
+
+कोड की दूसरी पंक्ति पाइपलाइन द्वारा उपयोग किए गए पूर्व-प्रशिक्षित मॉडल को डाउनलोड और कैश करती है, जबकि कोड की तीसरी पंक्ति दिए गए पाठ पर मूल्यांकन करती है। यहां उत्तर 99 आत्मविश्वास के स्तर के साथ "सकारात्मक" है।
+
+कई एनएलपी कार्यों में बॉक्स से बाहर पाइपलाइनों का पूर्व-प्रशिक्षण होता है। उदाहरण के लिए, हम किसी दिए गए पाठ से किसी प्रश्न का उत्तर आसानी से निकाल सकते हैं:
```
Comment (pacman100) [inline README_hd.md:126]: "out of the box" is translated in the literal sense which isn't quite understandable. Also, this deviates from the original readmedoc having vision example here

File: README_hd.md [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+
+
+## अधिक समझें
+
+|अध्याय | विवरण |
+|-|-|
+| [दस्तावेज़ीकरण](https://huggingface.co/transformers/) | पूरा एपीआई दस्तावेज़ीकरण और ट्यूटोरियल |
+| [कार्य सारांश](https://huggingface.co/docs/transformers/task_summary) | ट्रांसफॉर्मर समर्थित कार्य |
+| [प्रीप्रोसेसिंग ट्यूटोरियल](https://huggingface.co/docs/transformers/preprocessing) | मॉडल के लिए डेटा तैयार करने के लिए `टोकनाइज़र` का उपयोग करना |
+| [प्रशिक्षण और फाइन-ट्यूनिंग](https://huggingface.co/docs/transformers/training) | PyTorch/TensorFlow के ट्रेनिंग लूप या `ट्रेनर` API में ट्रांसफॉर्मर द्वारा दिए गए मॉडल का उपयोग करें |
```
Comment (pacman100) [inline README_hd.md:397]: here fine-tuning is written in Devanagari script which should be followed in other places above in the main content where it was translated as fixing

File: README_hd.md [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+| [क्विक स्टार्ट: ट्वीकिंग एंड यूज़ केस स्क्रिप्ट्स](https://github.com/huggingface/transformers/tree/main/examples) | विभिन्न कार्यों के लिए केस स्क्रिप्ट का उपयोग करें |
+| [मॉडल साझा करना और अपलोड करना](https://huggingface.co/docs/transformers/model_sharing) | समुदाय के साथ अपने ठीक-ठाक मॉडल अपलोड और साझा करें |
```
Comment (pacman100) [inline README_hd.md:399]: okay-okay as translation for fine-tuned

**Review by pacman100 — COMMENTED** [review]
Hello @AkshitGulyan, great work 🤗. Thanks for putting in a lot of effort for making  `transformers` accessible to an even larger number of people who are comfortable in Hindi language ✨. Looking good, left comments for improving readability.

File: README_hd.md [hunk 7]
```diff
@@ -0,0 +1,417 @@
...
+
+<h3 align="center">
+    <a href="https://hf.co/course"><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/course_banner.png"></a>
+</h3>
+
+🤗 Transformers 100 से अधिक भाषाओं में पाठ वर्गीकरण, सूचना निष्कर्षण, प्रश्न उत्तर, सारांशीकरण, अनुवाद, पाठ निर्माण का समर्थन करने के लिए हजारों पूर्व-प्रशिक्षित मॉडल प्रदान करता है। इसका उद्देश्य सबसे उन्नत एनएलपी तकनीक को सभी के लिए सुलभ बनाना है।
+
+🤗 Transformers त्वरित डाउनलोड और उपयोग के लिए एक एपीआई प्रदान करता है, जिससे आप किसी दिए गए पाठ पर एक पूर्व-प्रशिक्षित मॉडल ले सकते हैं, इसे अपने डेटासेट पर ठीक कर सकते हैं और इसे [मॉडल हब] (https://huggingface.co/models) के माध्यम से समुदाय के साथ साझा कर सकते हैं। ) . इसी समय, प्रत्येक परिभाषित पायथन मॉड्यूल पूरी तरह से स्वतंत्र है, जो संशोधन और तेजी से अनुसंधान प्रयोगों के लिए सुविधाजनक है।
+
+🤗 Transformers तीन सबसे लोकप्रिय गहन शिक्षण पुस्तकालयों का समर्थन करता है： [Jax](https://jax.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/) and [TensorFlow](https://www.tensorflow.org/) — और इसके साथ निर्बाध रूप से एकीकृत होता है। आप अपने मॉडल को सीधे एक ढांचे के साथ प्रशिक्षित कर सकते हैं और दूसरे के साथ लोड और अनुमान लगा सकते हैं।
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md:88]: yes it's a little confusing but i thought that i should translate each and every bit of the file !

File: README_hd.md [hunk 8]
```diff
@@ -0,0 +1,417 @@
...
+- [अनुवाद के लिए T5 का प्रयोग करें](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
+
+**[Write With Transformer](https://transformer.huggingface.co)**，हग फेस टीम द्वारा बनाया गया, यह एक आधिकारिक पाठ पीढ़ी है demo。
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md:103]: oops

File: README_hd.md [hunk 9]
```diff
@@ -0,0 +1,417 @@
...
+>>> tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+>>> model = TFAutoModel.from_pretrained("bert-base-uncased")
+
+>>> inputs = tokenizer("Hello world!", return_tensors="tf")
+>>> outputs = model(**inputs)
+```
+
+टोकननाइज़र सभी पूर्व-प्रशिक्षित मॉडलों के लिए प्रीप्रोसेसिंग प्रदान करता है और इसे सीधे एक स्ट्रिंग (जैसे ऊपर दिए गए उदाहरण) या किसी सूची पर बुलाया जा सकता है। यह एक डिक्शनरी (तानाशाही) को आउटपुट करता है जिसे आप डाउनस्ट्रीम कोड में उपयोग कर सकते हैं या `**` अनपैकिंग एक्सप्रेशन के माध्यम से सीधे मॉडल को पास कर सकते हैं।
+
+मॉडल स्वयं एक नियमित [Pytorch `nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) या [TensorFlow `tf.keras.Model`](https ://pytorch.org/docs/stable/nn.html#torch.nn.Module) ://www.tensorflow.org/api_docs/python/tf/keras/Model) (आपके बैकएंड के आधार पर), जो हो सकता है सामान्य तरीके से उपयोग किया जाता है। [यह ट्यूटोरियल](https://huggingface.co/transformers/training.html) बताता है कि इस तरह के मॉडल को क्लासिक PyTorch या TensorFlow प्रशिक्षण लूप में कैसे एकीकृत किया जाए, या हमारे `ट्रेनर` एपीआई का उपयोग कैसे करें ताकि इसे जल्दी से ठीक किया जा सके। एक नया डेटासेट।
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md:166]: @pacman100 
I really appreciate your contribution
Can you please edit the things which you are feeling that they should be formed in a different way !

File: README_hd.md [hunk 10]
```diff
@@ -0,0 +1,417 @@
...
+        <a href="https://github.com/huggingface/transformers/">English</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_zh-hant.md">繁體中文</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_ko.md">한국어</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_es.md">Español</a> |
+        <b>हिन्दी</b> |
+    <p>
+</h4>
+
+<h3 align="center">
+    <p>Jax, PyTorch और TensorFlow के लिए उन्नत प्राकृतिक भाषा संसाधन</p>
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md:88]: Fixed !

File: README_hd.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md [hunk 11]
```diff
@@ -0,0 +1,417 @@
...
+- [अनुवाद के लिए T5 का प्रयोग करें](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
+
+**[Write With Transformer](https://transformer.huggingface.co)**，हग फेस टीम द्वारा बनाया गया, यह एक आधिकारिक पाठ पीढ़ी है demo。
+
+## यदि आप हगफेस टीम से बीस्पोक समर्थन की तलाश कर रहे हैं
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md [hunk 12]
```diff
@@ -0,0 +1,417 @@
...
+    - सभी मॉडलों के लिए एकीकृत एपीआई
+
+1. कम कम्प्यूटेशनल ओवरहेड और कम कार्बन उत्सर्जन:
+    - शोधकर्ता हर बार नए सिरे से प्रशिक्षण देने के बजाय प्रशिक्षित मॉडल साझा कर सकते हैं
+    - इंजीनियर गणना समय और उत्पादन ओवरहेड को कम कर सकते हैं
+    - दर्जनों मॉडल आर्किटेक्चर, 2,000 से अधिक पूर्व-प्रशिक्षित मॉडल, 100 से अधिक भाषाओं का समर्थन
+
+1.मॉडल जीवनचक्र के हर हिस्से को शामिल करता है:
+    - कोड की केवल 3 पंक्तियों में उन्नत मॉडलों को प्रशिक्षित करें
+    - मॉडल को मनमाने ढंग से विभिन्न गहन शिक्षण ढांचे के बीच स्थानांतरित किया जा सकता है, जैसा आप चाहते हैं
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md [hunk 13]
```diff
@@ -0,0 +1,417 @@
...
+टोकननाइज़र सभी पूर्व-प्रशिक्षित मॉडलों के लिए प्रीप्रोसेसिंग प्रदान करता है और इसे सीधे एक स्ट्रिंग (जैसे ऊपर दिए गए उदाहरण) या किसी सूची पर बुलाया जा सकता है। यह एक डिक्शनरी (तानाशाही) को आउटपुट करता है जिसे आप डाउनस्ट्रीम कोड में उपयोग कर सकते हैं या `**` अनपैकिंग एक्सप्रेशन के माध्यम से सीधे मॉडल को पास कर सकते हैं।
+
+मॉडल स्वयं एक नियमित [Pytorch `nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) या [TensorFlow `tf.keras.Model`](https ://pytorch.org/docs/stable/nn.html#torch.nn.Module) ://www.tensorflow.org/api_docs/python/tf/keras/Model) (आपके बैकएंड के आधार पर), जो हो सकता है सामान्य तरीके से उपयोग किया जाता है। [यह ट्यूटोरियल](https://huggingface.co/transformers/training.html) बताता है कि इस तरह के मॉडल को क्लासिक PyTorch या TensorFlow प्रशिक्षण लूप में कैसे एकीकृत किया जाए, या हमारे `ट्रेनर` एपीआई का उपयोग कैसे करें ताकि इसे जल्दी से ठीक किया जा सके। एक नया डेटासेट।
+
+## ट्रांसफार्मर का उपयोग क्यों करें?
+
+1. उपयोग में आसानी के लिए उन्नत मॉडल:
+    - एनएलयू और एनएलजी पर बेहतर प्रदर्शन
+    - प्रवेश के लिए कम बाधाओं के साथ शिक्षण और अभ्यास के अनुकूल
+    - उच्च स्तरीय अमूर्तता, केवल तीन वर्गों को जानने की जरूरत है
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md [hunk 14]
```diff
@@ -0,0 +1,417 @@
...
+    - निर्बाध रूप से प्रशिक्षण, मूल्यांकन और उत्पादन के लिए सबसे उपयुक्त ढांचा चुनें
+
+1. आसानी से अनन्य मॉडल को अनुकूलित करें और अपनी आवश्यकताओं के लिए मामलों का उपयोग करें:
+    - हम मूल पेपर परिणामों को पुन: पेश करने के लिए प्रत्येक मॉडल आर्किटेक्चर के लिए कई उपयोग के मामले प्रदान करते हैं
+    - मॉडल की आंतरिक संरचना पारदर्शी और सुसंगत रहती है
+    - मॉडल फ़ाइल को अलग से इस्तेमाल किया जा सकता है, जो जादू संशोधन और त्वरित प्रयोग के लिए सुविधाजनक है
+
+## मुझे ट्रांसफॉर्मर का उपयोग कब नहीं करना चाहिए?
+
+- यह लाइब्रेरी मॉड्यूलर न्यूरल नेटवर्क टूलबॉक्स नहीं है। मॉडल फ़ाइल में कोड जानबूझकर अल्पविकसित है, बिना अतिरिक्त सार इनकैप्सुलेशन के, ताकि शोधकर्ता अमूर्तता और फ़ाइल जंपिंग में शामिल हुए बिना जादुई परिवर्तनों के माध्यम से जल्दी से पुनरावृति कर सकें।
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md [hunk 15]
```diff
@@ -0,0 +1,417 @@
...
+- `ट्रेनर` एपीआई किसी भी मॉडल के साथ संगत नहीं है, यह केवल इस पुस्तकालय के मॉडल के लिए अनुकूलित है। यदि आप सामान्य मशीन लर्निंग के लिए उपयुक्त प्रशिक्षण लूप कार्यान्वयन की तलाश में हैं, तो कहीं और देखें।
+- हमारे सर्वोत्तम प्रयासों के बावजूद, [उदाहरण निर्देशिका] (https://github.com/huggingface/transformers/tree/main/examples) में स्क्रिप्ट केवल उपयोग के मामले हैं। आपकी विशिष्ट समस्या के लिए, वे जरूरी नहीं कि बॉक्स से बाहर काम करें, और आपको कोड की कुछ पंक्तियों को सूट करने की आवश्यकता हो सकती है।
+
+## स्थापित करना
+
+### पाइप का उपयोग करना
```
[the remark this answers is not in the document]
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

File: README_hd.md (hunk 4 above)
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !
I wrote the out of the box in devanagari script,
Now i don't think anyone will find it confusing.

File: README_hd.md (hunk 5 above)
Comment (AkshitGulyan) [reply README_hd.md:397]: Fixed !

File: README_hd.md (hunk 6 above)
Comment (AkshitGulyan) [reply README_hd.md]: Fixed !

**Comment by AkshitGulyan** [comment]
Any Update ?

**Comment by sgugger** [comment]
I think you need to run `make fix-copies` on your side to adjust the READMEs, then it should be good to merge if all comments are addressed :-)

File: README_hd.md (hunk 1 above)
Comment (pacman100) [reply README_hd.md:207]: I meant Tensorflow and PyTorch has "स्थापना पृष्ठ" but Flax has "install page"

File: README_hd.md (hunk 2 above)
Comment (pacman100) [reply README_hd.md:212]: the cmd should be in english so that user can copy it and run?

File: README_hd.md (hunk 3 above)
Comment (pacman100) [reply README_hd.md:224]: the cmd should be in english so that user can copy it and run?

**Comment by pacman100** [comment]
Please address remaining comments and then we are good to go

File: README_hd.md (hunk 2 above)
Comment (AkshitGulyan) [reply README_hd.md:212]: Updated !

File: README_hd.md (hunk 3 above)
Comment (AkshitGulyan) [reply README_hd.md]: Updated !

File: README_hd.md (hunk 1 above)
Comment (AkshitGulyan) [reply README_hd.md]: Done !

**Comment by AkshitGulyan** [comment]
> Please address remaining comments along with steps Sylvain has mentioned and then we are good to go

Addressed all the comments and updated the file according to them !
Can you please help me understanding this fix-copies concept which Sylvain has mentioned as i dont know about it !

**Comment by pacman100** [comment]
Hello @AkshitGulyan, in the above PR I fixed subtle and time-consuming bugs to run `make fix-copies` without any issues. The details are below so that you can do these things next time.

1. When I ran `make fix-copies` locally I got below error:
```
(ml) sourabmangrulkar@Sourabs-MacBook-Pro transformers % make fix-copies
python utils/check_copies.py --fix_and_overwrite
Traceback (most recent call last):
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 572, in <module>
    check_copies(args.fix_and_overwrite)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 270, in check_copies
    check_model_list_copy(overwrite=overwrite)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 455, in check_model_list_copy
    localized_md_list = get_model_list(filename, _start_prompt, _end_prompt)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 303, in get_model_list
    while not lines[start_index].startswith(start_prompt):
IndexError: list index out of range
make: *** [fix-copies] Error 1
```  

2. After spending time diving into `utils/check_copies.py` found the issue wherein `prompt_start` specified was not matching to the line in `README_hd.md`. Made them same.

3. Then got this issue:
```
Traceback (most recent call last):
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 354, in convert_to_localized_md
    localized_model_index = {
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 355, in <dictcomp>
    re.search(r"\*\*\[([^\]]*)", line).groups()[0]: line
AttributeError: 'NoneType' object has no attribute 'groups'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 575, in <module>
    check_copies(args.fix_and_overwrite)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 270, in check_copies
    check_model_list_copy(overwrite=overwrite)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 459, in check_model_list_copy
    readmes_match, converted_md_list = convert_to_localized_md(md_list, localized_md_list, _format_model_list)
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 359, in convert_to_localized_md
    raise AttributeError("A model name in localized READMEs cannot be recognized.")
AttributeError: A model name in localized READMEs cannot be recognized.
(ml) sourabmangrulkar@Sourabs-MacBook-Pro transformers % python utils/check_copies.py
Traceback (most recent call last):
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 351, in convert_to_localized_md
    localized_model_index = {
  File "/Users/sourabmangrulkar/Code/transformers/utils/check_copies.py", line 352, in <dictcomp>
    re.search(r"\*\*\[([^\]]*)", line).groups()[0]: line
AttributeError: 'NoneType' object has no attribute 'groups'
```

This was a subtle bug which took quite some time to figure out. You had improperly formatted the following models with improper spaces resulting in regex failing, below shows the buggy version:
```
1. ** [TrOCR] (https://huggingface.co/docs/transformers/model_doc/trocr) ** (from Microsoft) released with the paper [TrOCR: Transformer-based Optical Character Recognition with Pre-trained Models](https://arxiv.org/abs/2109.10282) by Minghao Li, Tengchao Lv, Lei Cui, Yijuan Lu, Dinei Florencio, Cha Zhang, Zhoujun Li, Furu Wei.
1. ** [UL2] (https://huggingface.co/docs/transformers/model_doc/ul2) ** (from Google Research) released with the paper [Unifying Language Learning Paradigms](https://arxiv.org/abs/2205.05131v1) by Yi Tay, Mostafa Dehghani, Vinh Q. Tran, Xavier Garcia, Dara Bahri, Tal Schuster, Huaixiu Steven Zheng, Neil Houlsby, Donald Metzler 
```

So, after fixing it everything works as expected:
```
(ml) sourabmangrulkar@Sourabs-MacBook-Pro transformers % make fix-copies             
python utils/check_copies.py --fix_and_overwrite
python utils/check_table.py --fix_and_overwrite
python utils/check_dummies.py --fix_and_overwrite
```

Also, many model list is very very inconsistent with some having names in Hindi while others in English. Follow the format where all model names are in latin script instead of Devanagari script.

**Comment by pacman100** [comment]
Hello @AkshitGulyan, please transfer the changes from above sample PR to this PR. Thank you and hope the above explanation clarifies the steps that Sylvain was suggesting.

**Pull request closed without merging**